addons/wowl/doc/systray.md

```diff
@@ -15,8 +15,9 @@ A systray item is an object with the following properties:
   usually prefixed by the current module name)
 - `Component`: the Component class that will be used to display the item. Its root
   node has to be a `<li>` tag!
-- `sequence (number, optional)`: if given, this number will be used to order the
-  items.
+- `sequence (number, optional)`: defaults to 50. If given, this number will be used
+  to order the items. The lowest priority is on the left and the highest priority
+  is on the right in the systray menu.
```

Comment:
I think it's the opposite :D

Reply:
Sequence 0 = hight priority => right 
Sequence 100 = low priority => left no ?
It s maybe more clear to use Sequence 

- `sequence (number, optional)`: defaults to 50. If given, this number will be used
   to order the items. The lowest sequence is on the right and the highest sequence
   is on the left in the systray menu.

Reply:
Yep, that's why I suggested to use 'sequence'.